### PR TITLE
Update zh_cn.json

### PR DESCRIPTION
**Pull request by Neubulae**
...and some reformatting

~~Yeah I'm here to save the world again~~

### CODE REVIEW

**Comment by RaymondBlaze** [comment]
In the first place, thanks for your work!

Here are some suggestions from my POV:
1. About removing "You"s: I can't see the meaning in doing so. I would rather believe it is not a good idea to ignore gramma components in the original text. From my POV, the usage of second person POV in the original text should be considered as a kind of **STYLE**, so there's no good ignoring it.
2. About Phantom: It's quite obvious that the idea of phasing is not directly originated from the mob so called Phantom. Instead, it's more likely originated from the word "Phantom" itself. So I don't think it's a good idea to simply use the mob's Chinese name in translating the word "phantomize".
3. About Unwieldly: It's quite weird that you stick to the original text again in the translation of this word (as in suggestion 1 you didn't show such respect to them when removing "You"s). Surely, Unwieldly means one can not hold something, but it can be referred from the description that this "something" means shield. So I would rather not to change the previous translation to  yours.
4. About Hunger and Saturation: I noticed that you're using a deprecated translation ”饱食度“. Minecraft official Chinese translation has had deprecated this word for years, I don't think it's a good idea to bring it back. It can be referred from common sense that it's 饥饿值 (Hunger Value) that restricts sprinting.

It's no good to just make critics, so I've decided create my translation. Later I'll make an PR.

**Comment by Neubulae** [comment]
> In the first place, thanks for your work!
> 
> Here are some suggestions from my POV:
> 
>     1. About removing "You"s: I can't see the meaning in doing so. I would rather believe it is not a good idea to ignore gramma components in the original text. From my POV, the usage of second person POV in the original text should be considered as a kind of **STYLE**, so there's no good ignoring it.
> 
>     2. About Phantom: It's quite obvious that the idea of phasing is not directly originated from the mob so called Phantom. Instead, it's more likely originated from the word "Phantom" itself. So I don't think it's a good idea to simply use the mob's Chinese name in translating the word "phantomize".
> 
>     3. About Unwieldly: It's quite weird that you stick to the original text again in the translation of this word (as in suggestion 1 you didn't show such respect to them when removing "You"s). Surely, Unwieldly means one can not hold something, but it can be referred from the description that this "something" means shield. So I would rather not to change the previous translation to  yours.
> 
>     4. About Hunger and Saturation: I noticed that you're using a deprecated translation ”饱食度“. Minecraft official Chinese translation has had deprecated this word for years, I don't think it's a good idea to bring it back. It can be referred from common sense that it's 饥饿值 (Hunger Value) that restricts sprinting.
> 
> 
> It's no good to just make critics, so I've decided create my translation. Later I'll make an PR.

关于“你不你”的问题，我是觉得中文语境下一串非常descriptive的文本理应尽可能减少“你”的使用，换句话说我是觉得“你”少点对于中文母语着来讲更加自然。
我之所以选用“幻翼化”这样的字眼是因为我印象里当初在游玩过程中意识到了“幻化”这种字眼实际上给我的游戏体验带来了一定程度的负面影响（也就是说我玩下来一趟以后认为幻翼化比幻化要更合适一些）。现在仔细想想好像不大必要，可以采纳。
至于“持盾不能”我其实是**有意去除**掉的（印象里这个原本也是我提交的？），一来是不好听，二来是考虑原文作为一个轻佻的字眼似乎不大适合直接写破。
翻译的deprecation我竟然不知道！awsl

**Comment by Neubulae** [comment]
还有我这个是有意按照英文json格式化的，免得以后对照工作又出问题……

**Comment by RaymondBlaze** [comment]
> 关于“你不你”的问题，我是觉得中文语境下一串非常descriptive的文本理应尽可能减少“你”的使用，换句话说我是觉得“你”少点对于中文母语着来讲更加自然

我也倾向于尽量减少”你“在同一句内的多次使用和不应该在描述性文本里用第二人称这点，但是原文如此，我感觉完全从一句话里面去除掉”你“的话稍微有点不太尊重原文风格和原有的句子成分的意思。而且在其他句子实在不便于彻底去掉”你“的使用的情况下，只去掉部分句子中的”你“会导致一定程度的风格割裂。还有一点就是，你这个PR里去除”你“字之后留下的句子似乎有的读起来不太通顺。

> 我之所以选用“幻翼化”这样的字眼是因为我印象里当初在游玩过程中意识到了“幻化”这种字眼实际上给我的游戏体验带来了一定程度的负面影响（也就是说我玩下来一趟以后认为幻翼化比幻化要更合适一些）。

这个我质疑的角度主要是幻翼实际上并没有穿墙能力（而幻影族也并没有幻翼的飞行能力），从一开始翻译成幻影族的时候其实就已经意识到Apace设计这个种族的特性的时候不是出于对幻翼的完全模仿。穿墙更像是来源自Phantom一词的本义。

> 至于“持盾不能”我其实是有意去除掉的（印象里这个原本也是我提交的？），一来是不好听，二来是考虑原文作为一个轻佻的字眼似乎不大适合直接写破。

这个我部分同意，不过”把持“或者”把握“在中文里引申义多少有点多，使用起来我怕有一定歧义。

其实还有一些琐碎的小问题是我上面没有提到的：比如”脆皮“这种过于口语化的词汇的使用，种族名也是值得商榷的（要么就彻底差异化，要么就保持统一的后缀，原本的蛛形纲和猫科这样使用生物学名词的真的有点画风不对了）

**Comment by Neubulae** [comment]
> > 关于“你不你”的问题，我是觉得中文语境下一串非常descriptive的文本理应尽可能减少“你”的使用，换句话说我是觉得“你”少点对于中文母语着来讲更加自然
> 
> 我也倾向于尽量减少”你“在同一句内的多次使用和不应该在描述性文本里用第二人称这点，但是原文如此，我感觉完全从一句话里面去除掉”你“的话稍微有点不太尊重原文风格和原有的句子成分的意思。而且在其他句子实在不便于彻底去掉”你“的使用的情况下，只去掉部分句子中的”你“会导致一定程度的风格割裂。还有一点就是，你这个PR里去除”你“字之后留下的句子似乎有的读起来不太通顺。
> 
> > 我之所以选用“幻翼化”这样的字眼是因为我印象里当初在游玩过程中意识到了“幻化”这种字眼实际上给我的游戏体验带来了一定程度的负面影响（也就是说我玩下来一趟以后认为幻翼化比幻化要更合适一些）。
> 
> 这个我质疑的角度主要是幻翼实际上并没有穿墙能力（而幻影族也并没有幻翼的飞行能力），从一开始翻译成幻影族的时候其实就已经意识到Apace设计这个种族的特性的时候不是出于对幻翼的完全模仿。穿墙更像是来源自Phantom一词的本义。
> 
> > 至于“持盾不能”我其实是有意去除掉的（印象里这个原本也是我提交的？），一来是不好听，二来是考虑原文作为一个轻佻的字眼似乎不大适合直接写破。
> 
> 这个我部分同意，不过”把持“或者”把握“在中文里引申义多少有点多，使用起来我怕有一定歧义。
> 
> 其实还有一些琐碎的小问题是我上面没有提到的：比如”脆皮“这种过于口语化的词汇的使用，种族名也是值得商榷的（要么就彻底差异化，要么就保持统一的后缀，原本的蛛形纲和猫科这样使用生物学名词的真的有点画风不对了）

有什么好的意见可以在Files Changed按加号提出修改意见呀，我觉得你想法挺好（

**Comment by RaymondBlaze** [comment]
> 有什么好的意见可以在Files Changed按加号提出修改意见呀，我觉得你想法挺好（

这点一开始不是没考虑过，但是在我看了一眼当前的汉化发现那里面也有一些我认为需要修改的而你没有修改的……
所以我干脆把我那份PR到你那边做个对比吧，这样更明显一点。而且我已经和你一样把顺序整到和英文那边一致了，应该会方便一点？

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 1]
```diff
@@ -1,125 +1,199 @@
...
-    "origin.origins.merling.name":"人鱼族",
-    "origin.origins.merling.description":"海洋的原住民。他们并不习惯长时间地离开水体。",
+    "origin.origins.human.description":"普通的人类，普通的Minecraft体验。",
+
+    "origin.origins.merling.name":"人鱼",
+    "origin.origins.merling.description":"海洋的原住民。他们并不习惯长时间离开水体。",
+
     "origin.origins.arachnid.name":"蛛形纲",
-    "origin.origins.arachnid.description":"运用蜘蛛网诱捕敌人是他们的专长，依附攀爬也是他们的强项——这使它们成为完美的猎手。",
+    "origin.origins.arachnid.description":"运用蜘蛛网诱捕敌人，依附攀岩是他们作为完美猎人的天生专长。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:9]: 原文："Their **climbing** abilities and the ability to trap their foes in spiderweb **make** the Arachnid perfect hunters."
1. make指示的因果关系没了
2. climbing译作“攀**岩**”而非“攀爬”不太合适，攀岩是一项具体的运动，攀爬才是一种普遍的能力

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 2]
```diff
@@ -1,125 +1,199 @@
...
-    "origin.origins.phantom.description":"作为人与幻翼的混血后代，这些生物可以幻化自如。",
+    "origin.origins.phantom.description":"作为人与幻翼的混血后代的他们可以自如幻化。",
+
     "origin.origins.feline.name":"猫科",
-    "origin.origins.feline.description":"猫科的外表足以吓跑苦力怕。灵活的动作使他们总能安全着陆。",
-    "origin.origins.elytrian.name":"翼人族",
-    "origin.origins.elytrian.description":"翼人族擅长随风飘泊。如果头顶的空间不够宽阔，他们会感到不舒服。",
-    "origin.origins.enderian.name":"末影族",
-    "origin.origins.enderian.description":"作为末影龙的子嗣，末影族能够随意传送，但会受到水的伤害。",
+    "origin.origins.feline.description":"猫科动物的外表正是苦力怕的原始恐惧。擅长灵活机动也总能安全着陆。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:21]: 原文："With their cat-like appearance, the Feline scare creepers away. With the dexterity of cats, they always land safely on their feet."
自由发挥成分是不是太大了一点……“原始”这种无中生有的成分暂且按下不谈，第二句的主语和with指示的因果关系完全被你抛弃了

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 3]
```diff
@@ -1,125 +1,199 @@
...
+
+    "origin.origins.elytrian.name":"翼人",
+    "origin.origins.elytrian.description":"翼人喜欢随风飘荡，在广袤的天穹下翱翔。因此如果头顶的空间不够宽阔，他们会感到不适。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:24]: 原文："Often flying around in the winds, Elytrians are uncomfortable when they don't have enough space above their head."
不太清楚这个“在广袤的天穹下翱翔”从何而来……原译的“随风飘荡”已经把“Often flying around in the winds”的意思表达出来了

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 4]
```diff
@@ -1,125 +1,199 @@
...
+
+    "origin.origins.enderian.name":"末影后裔",
+    "origin.origins.enderian.description":"末影龙的子嗣传承了传送的能力，但同样会受到水的伤害。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:27]: 原文："Born as children of the Ender Dragon, Enderians are capable of teleporting but are vulnerable to water."
没看到“传承”一词在原文中的对应，而且末影龙也没有传送能力（虽说Apace的原设定就挺怪，末影龙的后裔为什么不是龙裔XD）。“**同样**会受到水的伤害”也是无中生有的，末影龙不会被水伤害。
种族名格式这事我还是倾向于统一和彻底差异化二选一，前者成本较低（毕竟为每个种族都做出修改不一定能让玩家接受）。而且“末影后裔”并不是特别顺口，如果是要把“Dragonborn”的起源翻译成“龙裔”倒是还比较能接受。

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 5]
```diff
@@ -1,125 +1,199 @@
...
+    
     "origin.origins.shulk.name":"潜影族",
-    "origin.origins.shulk.description":"潜影贝的近亲, 体表覆盖着一层保护性的壳状皮肤。",
+    "origin.origins.shulk.description":"潜影贝的近亲, 体表覆有一层保护性的甲壳皮肤。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:30]: 原文："Related to Shulkers, the bodies of the Shulk are outfitted with a protective shell-like skin."
shell-like -> 壳状，没问题吧，没看出修改的必要

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 6]
```diff
@@ -1,125 +1,199 @@
...
-    "power.origins.like_water.description":"你在水下的时候不会下沉，除非你想。",
-    "power.origins.fragile.name":"脆弱",
-    "power.origins.fragile.description":"你的生命上限少了3颗心。",
-    "power.origins.webbing.name":"网织",
-    "power.origins.webbing.description":"当你以近战击中一个敌人时，他们会被困在蛛网里。",
-    "power.origins.climbing.name":"攀爬",
-    "power.origins.climbing.description":"你可以攀上任何一面墙，而不仅仅是梯子。",
-    "power.origins.carnivore.name":"食肉",
-    "power.origins.carnivore.description":"你的只吃肉，不吃蔬菜。",
+    "power.origins.like_water.description":"你在水下不会沉底，除非你想。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:45]: 下沉和沉底的区别还是挺大的，人鱼族可以在任意水深都保持静止的，还是下沉能描述的比较好

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 7]
```diff
@@ -1,125 +1,199 @@
...
+
+    "power.origins.fragile.name":"脆皮",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:47]: 过于口语化

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 8]
```diff
@@ -1,125 +1,199 @@
...
+    "power.origins.fragile.description":"你的生命上限少3颗心。",
+
+    "power.origins.webbing.name":"织网专精",
+    "power.origins.webbing.description":"近战击中敌人时，它会被困在蛛网里。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:51]: 比较神奇的是，原文用的是“they”
顺便我不太赞同往能力名里添油加醋的加入“专精”“能人”“动物”这样原文没有的成分，能力名并不需要一定是个偏正短语

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 9]
```diff
@@ -1,125 +1,199 @@
...
+
+    "power.origins.climbing.name":"攀岩专精",
+    "power.origins.climbing.description":"你可以攀上任何一面墙，不仅仅是梯子。",
+
+    "power.origins.carnivore.name":"肉食动物",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:56]: 食肉，肉食动物，肉食性我在你的repo那边的PR解释了一下该选哪个

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 10]
```diff
@@ -1,125 +1,199 @@
...
+    "power.origins.carnivore.description":"你只能吃肉。",
+
     "power.origins.fire_immunity.name":"火焰免疫",
     "power.origins.fire_immunity.description":"你免疫所有类型的火焰伤害。",
-    "power.origins.nether_spawn.name":"下界居民",
-    "power.origins.nether_spawn.description":"你的出生点将位于下界。",
+
+    "power.origins.nether_spawn.name":"下界原住民",
+    "power.origins.nether_spawn.description":"你的出生点位于下界。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:63]: 把原文的will直接无视似乎不太好吧，这些描述最重要的出现场合是第一次进入世界前的选择界面，将来时是合理的

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 11]
```diff
@@ -1,125 +1,199 @@
...
+
     "power.origins.burning_wrath.name":"怒火",
     "power.origins.burning_wrath.description":"着火时，你的攻击造成额外伤害。",
+
     "power.origins.hotblooded.name":"炽血",
-    "power.origins.hotblooded.description":"你灼热的身体使毒液快速挥发，因此你对中毒和饥饿药水效果免疫。",
+    "power.origins.hotblooded.description":"灼热的体温快速蒸腾毒液，使你免疫中毒和饥饿药效。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:69]: 虽然在另一边解释过了不过还是再说明一下吧：原文用的是burn，挥发和蒸腾可能都不太合适；status effect也有官译“状态效果”

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 12]
```diff
@@ -1,125 +1,199 @@
...
     "power.origins.slow_falling.description":"你会像羽毛一样轻轻地飘落到地上，除非你潜行。",
-    "power.origins.vegetarian.name":"食草动物",
+
+    "power.origins.vegetarian.name":"草食动物",
     "power.origins.vegetarian.description":"你消化不了肉。",
-    "power.origins.fresh_air.name":"新鲜空气",
-    "power.origins.fresh_air.description":"睡觉时，你的床必须处于至少86格的高度，这样才能呼吸新鲜空气。",
+
+    "power.origins.fresh_air.name":"新风需求",
+    "power.origins.fresh_air.description":"你必须在至少86格高的床上睡眠，才能呼吸新鲜空气。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:84]: 相比于“床处于86格的高度”，“86格高的床”是不是会造成比较离谱的歧义？<del>一张宽1格长2格高86格的床</del>
不过新风需求这个翻译我很喜欢 :)

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 13]
```diff
@@ -1,125 +1,199 @@
...
+
+    "power.origins.hunger_over_time.name":"高速代谢",
+    "power.origins.hunger_over_time.description":"幻翼化会让你感到饥饿。",
+
+    "power.origins.burn_in_daylight.name":"光过敏",
     "power.origins.burn_in_daylight.description":"如果不隐形，你会被日光点燃。",
-    "power.origins.fall_immunity.name":"杂技高手",
-    "power.origins.fall_immunity.description":"你不会受到摔落伤害，无论从何处跌落。",
+
+    "power.origins.fall_immunity.name":"体操能手",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:98]: 和上面的攀岩的问题类似，把范围限定在运动范围内多少有些奇怪？

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 14]
```diff
@@ -1,125 +1,199 @@
...
+    "power.origins.fall_immunity.description":"你免疫摔落伤害。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:99]: 原文虽然有废话的嫌疑，但是直接删除还是不太符合翻译原则的

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 15]
```diff
@@ -1,125 +1,199 @@
...
+    "power.origins.aerial_combatant.description":"鞘翅滑翔会带来巨额伤害增幅。",
+
     "power.origins.throw_ender_pearl.name":"传送",
-    "power.origins.throw_ender_pearl.description":"只要你愿意，你可以抛出一个不造成伤害的末影珍珠。",
-    "power.origins.pumpkin_hate.name":"南瓜恐惧",
-    "power.origins.pumpkin_hate.description":"你害怕南瓜，理由想必很充分。",
+    "power.origins.throw_ender_pearl.description":"你随时可以抛出一个无伤的末影珍珠。",
+
+    "power.origins.pumpkin_hate.name":"南瓜恐惧症",
+    "power.origins.pumpkin_hate.description":"你不可抗拒而本能地恐惧南瓜。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:138]: 原文的"For a good reason"不知所踪

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 16]
```diff
@@ -1,125 +1,199 @@
...
     "power.origins.extra_reach.name":"瘦长身体",
     "power.origins.extra_reach.description":"你可以触及更远的方块和实体。",
-    "power.origins.shulker_inventory.name":"贮藏家",
+
+    "power.origins.shulker_inventory.name":"贮藏大师",
     "power.origins.shulker_inventory.description":"你可以多带9个物品，这9个新增的槽位死亡不掉落。",
+
     "power.origins.natural_armor.name":"坚韧皮肤",
-    "power.origins.natural_armor.description":"你的皮肤天生予以你保护，即使不穿铠甲。",
+    "power.origins.natural_armor.description":"你的皮肤天生予以保护，哪怕不穿铠甲。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:147]: 没有看出省略掉第二个你的意义，句子都变得不通顺了
把书面语“即使”改为口语“哪怕”也是意义不明的

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 17]
```diff
@@ -1,125 +1,199 @@
...
     "key.origins.view_origin":"查看起源",
+
     "text.autoconfig.origins.title":"起源设置",
     "text.autoconfig.origins.option.xOffset":"冷却条 X轴偏移量",
     "text.autoconfig.origins.option.yOffset":"冷却条 Y轴偏移量",
-    "text.autoconfig.origins.option.phantomizedOverlayStrength":"幻化时视觉效果强度",
-    "item.origins.orb_of_origin":"起源之球",
+    "text.autoconfig.origins.option.phantomizedOverlayStrength":"幻化时的视觉效果强度",
+
+    "item.origins.orb_of_origin":"起源宝珠",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:189]: orb我查了牛津，韦氏，柯林斯，都没有发现“宝珠”这一释义，唯一相近的是“国王或女王在重要仪式上携带的**顶部装饰有十字架**的宝球”，显然和这个物品没有任何关系。原本的翻译直接用orb最常见的释义“球，球状物”没啥问题。

File: src/main/resources/assets/origins/lang/zh_cn.json [hunk 18]
```diff
@@ -133,35 +207,32 @@
...
-    "origins.gui.badge.toggle": "Toggle with %s.",
-    "origins.gui.unbound_key": "§ounbound§r",
-    "commands.origin.get.result": "%s has the following %s: %s (%s)",
-    "commands.origin.gui.all": "Opened the selection GUI for %s players.",
-    "commands.origin.gui.layer": "Opened the \"%2$s\" selection GUI for %1$s players."
+    "commands.origin.set.success.single":"已将%s的%s层起源设置为：%s。",
+    "commands.origin.set.success.multiple":"已将%s个目标的%s层起源设置为：%s。",
+    "commands.origin.get.result":"%s的%s是：%s（%s）",
+    "commands.origin.gui.all":"已为%s玩家打开选择界面。",
+    "commands.origin.gui.layer":"已为%1$s玩家打开\"%2$s\"选择界面。",
```
Comment (RaymondBlaze) [inline src/main/resources/assets/origins/lang/zh_cn.json:235]: 这里的%s应该是执行成功的玩家数，所以应该是%s**个**玩家
后面的转义符转换后是英文双引号，按中文的习惯直接改为中文双引号或者空格即可

**Comment by Neubulae** [comment]
最近在忙期中事物，我先draft掉，实在抱歉……

For the time being, with massive irl things™ on hand, I drafted this PR planning to work on improvements suggested by @LimonBlaze later after fixing my irl issues, sorry for the bikeshedderities™!

**Comment by apace100** [comment]
Closing this in favor of #513 which is more recent - I assume it contains the results of your discussions here.
Feel free to create a new PR if you want to improve the translation further. :)
Thanks for your work!

**Pull request closed without merging**